docs/source/conf.py

```diff
@@ -178,3 +180,14 @@
     # https://github.com/sphinx-doc/sphinx/issues/4961
     "ref.python",
 ]
+
+
+def do_not_skip_os_modules(app: Sphinx, what: str, name: str, obj, skip: bool, options: list[str]) -> bool:
+    if name.endswith("._os") and what == "module":
```

Comment:
```suggestion
def autoapi_skip_hook(app: Sphinx, what: str, name: str, obj, skip: bool, options: list[str]) -> bool:
    # Do not skip OS modules in dissect.target (caught by `private-members`)
    if name.endswith("._os") and what == "module":
```
Maybe rename to something more generic in case we want to expand this.